landoapi/workers/landing_worker.py

```diff
@@ -439,6 +439,11 @@ def run_job(
             LandingJobAction.LAND, commit_id=commit_id, commit=True, db=db
         )
 
+        # Patches are no longer necessary, delete them.
+        for revision in job.revisions:
+            revision.patch_bytes = b""
+        db.session.commit()
+
```

Comment:
We discussed this previously and concluded that since the commit hash is present, then it's not going to be much added value to have the raw patch after landing (for successful landings, at least).

I agree that there could be some benefit to keeping those around, however, these are taking a bit more than a few MBs at this time (a good portion of the 2.3GB as of today) and appear to be increasing dramatically in the last few weeks. As I understand we have a cap of 10GB (or at least, in the current tier).

The time limit approach might be better though regardless. Let me think some more about this.